hadoop-ozone/dist/src/main/smoketest/httpfs/operations.robot

```diff
@@ -17,6 +17,7 @@
 Library             Process
 Library             BuiltIn
 Library             String
+Suite Setup         Get Security Enabled From Config
```

Comment:
`operations.robot` does not have test cases (it's a resource file), so this can be removed.